src/pages/console/formbuilder/call-to-action.mdx

```diff
@@ -1,9 +1,10 @@
 export const meta = {
-  title: "Customize action buttons (Submit, Cancel, Clear, Reset)",
-  description: "Amplify Studio's generated forms come with three action buttons: **Submit**, **Cancel**, and **Clear** or **Reset**, depending on whether the form creates or updates a record."
-}
+  title: 'Customize action buttons (Submit, Cancel, Clear, Reset)',
+  description:
+    "Amplify Studio's generated forms come with three action buttons: **Submit**, **Cancel**, and **Clear** or **Reset**, depending on whether the form creates or updates a record."
+};
 
-Amplify Studio's generated forms come with three action buttons: **Submit**, **Cancel**, and **Clear** or **Reset**, depending on whether the form creates or updates a record.
+[Amplify Studio's](/console/adminui/start/#logging-in-and-creating-an-app) generated forms come with three action buttons: **Submit**, **Cancel**, and **Clear** or **Reset**, depending on whether the form creates or updates a record.
```

Comment:
Rather than linking to the login at the first mention of Studio, I recommend moving the link to the first instances of working inside Studio